lab-andrew/lib/threecontrol.js

```diff
@@ -0,0 +1,10 @@
+'use strict';
```

Comment:
This is also great, but you're basically just repeating your code from onecontrol.js.  You should create a module that does what these three functions do, but also takes a parameter to determine what file it should look up.  For example something like this, readFile(file, callback) { ... }.  That way you'll be following DRY methodology.